### DISCUSSION TITLE
Jellyfin

### DISCUSSION THREAD

**Discussion by burnclouds** [nextcloud/all-in-one#3500] [Ideas] [answered]
I see more community containers such as caddy and plex being added.
Can one for Jellyfin be added. I don't do much in github but I'm trying to migrate from plex to jellyfin. having only recently migrated from ncp to all-in-one. Seems like adding caddy, plex, and jellyfin community containers would be perfect for me. what considerations am i forgetting? What would it take to get a community container for jellyfin addeed? https://jellyfin.org/docs/general/networking/caddy/

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-add-containers

Regarding the json, I just leave this here as an example:

```
{
    "aio_services_v1": [
        {
            "container_name": "nextcloud-aio-jellyfin",
            "display_name": "Jellyfin",
            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/jellyfin",
            "image": "jellyfin/jellyfin",
            "image_tag": "latest",
            "internal_port": "host",
            "restart": "unless-stopped",
            "environment": [
                "TZ=%TIMEZONE%",
            ],
            "volumes": [
                {
                    "source": "nextcloud_aio_jellyfin",
                    "destination": "/config",
                    "writeable": true
                },
                {
                    "source": "%NEXTCLOUD_DATADIR%",
                    "destination": "/media",
                    "writeable": false
                },
                {
                    "source": "%NEXTCLOUD_MOUNT%",
                    "destination": "%NEXTCLOUD_MOUNT%",
                    "writeable": false
                }
            ],
            "devices": [
                "/dev/dri"
            ],
            "backup_volumes": [
                "nextcloud_aio_jellyfin"
            ]
        }
    ]
}
```

**Reply by szaimen** [reply]
@burnclouds would you be up for creating the PR that adds this?

**Reply by burnclouds** [reply]
I don't have an environment set up to do that right now.

**Reply by szaimen** [reply]
Alll right. Basically you would only need to create a PR which adds the needed files. Is this what you are not able to do currently?

**Reply by burnclouds** [reply]
Correct

**Reply by szaimen** [reply]
okay

**Reply by burnclouds** [reply]
I don't use github all too often. But I think I successfully created a PR for this.
https://github.com/nextcloud/all-in-one/pull/3634

**Reply by szaimen** [reply]
Hi @burnclouds have you lost the interest in adding this?

**Reply by AiroPi** [reply]
> Hi @burnclouds have you lost the interest in adding this?

Hi, I'm interested in Jellyfin. Since it appears that burnclouds is not active in this community container idea, would it be okay if I re-open the PR? I will mention burnclouds because he did most of the work, even though he didn't finish it.

I am just wondering how I can test the community-container on my side before publishing it? Because aio fetches the containers from the GitHub repository by name, so they need to be already present, right?

**Reply by szaimen** [reply]
Yes, of course, feel free to take over :)

You could for testing mount the folder that you are creating in `/var/www/docker-aio/community-containers/jellyfin` inside the mastercontainer which should allow to test it locally already. Apart from that, I will also publish it to the develop channel and beta channel which you can use for testing later on.

**Reply by szaimen** [reply, marked as answer]
This is now released with v8.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
